ParabolInc/parabol
# feat: Improve clarity of Integrations page

Fixes #5875
Fixes #6797

<img width="791" alt="image" src="https://user-images.githubusercontent.com/466991/175012964-48e44e64-f0cc-4ca2-bc0f-6bd3a6192bff.png">

**How to test:**

- Make all integrations visible (Add feature flags for Azure DevOps and MS Teams, add Azure DevOps provider)
- Check that all integrations can be connected and all controls are working as expected
- Check mobile markup
